.github/workflows/ci_initiatives.yml

```diff
@@ -82,7 +82,7 @@ jobs:
       - run: bundle exec brakeman --rails6 --force-scan .
         name: Scanning Security issues on module
         working-directory: ${{ env.DECIDIM_MODULE }}
-      - run: bundle exec rspec
+      - run: bundle exec rspec --exclude-pattern 'spec/system/**/*_spec.rb'
```

Comment:
Let's change the name of this workflow to `"[CI] Initiatives (unit tests)"` so it's consistent